data/json/mapgen/homeimprovement_superstore_new.json

```diff
@@ -1,120 +1,4 @@
 [
-  {
-    "id": "hardware_clothing",
-    "type": "item_group",
-    "items": [
-      [ "hat_hard", 100 ],
-      [ "boots", 50 ],
-      [ "glasses_safety", 50 ],
-      [ "tool_belt", 25 ],
-      [ "elbow_pads", 50 ],
-      [ "knee_pads", 50 ],
-      [ "gloves_work", 45 ],
-      [ "ear_plugs", 75 ],
-      [ "mask_dust", 75 ],
-      [ "miner_hat", 5 ],
-      [ "cleansuit", 5 ]
-    ]
-  },
-  {
-    "id": "vending_drink_home_impr",
-    "type": "item_group",
-    "items": [
-      [ "water_clean", 50 ],
-      [ "water_mineral", 20 ],
-      [ "oj", 40 ],
-      [ "juice", 35 ],
-      [ "cranberry_juice", 35 ],
-      [ "cola", 50 ],
-      [ "lemonlime", 45 ],
-      [ "V8", 15 ],
-      [ "gum", 15 ]
-    ]
-  },
-  {
-    "id": "hardware_trash",
-    "type": "item_group",
-    "items": [
-      [ "wrapper", 50 ],
-      [ "bottle_glass", 50 ],
-      [ "bottle_plastic", 70 ],
-      [ "can_drink_unsealed", 60 ],
-      [ "flyer", 30 ],
-      [ "box_small", 30 ],
-      [ "rag", 20 ],
-      [ "bag_plastic", 20 ],
-      [ "news_regional", 15 ],
-      [ "mag_news", 15 ]
-    ]
-  },
-  {
-    "id": "paint",
-    "type": "item_group",
-    "items": [
-      [ "r_paint", 45 ],
-      [ "b_paint", 45 ],
-      [ "w_paint", 45 ],
-      [ "g_paint", 45 ],
-      [ "p_paint", 45 ],
-      [ "y_paint", 45 ],
-      [ "paint_brush", 55 ]
-    ]
-  },
-  {
-    "id": "carpet",
```

Comment:
looks like you nuked the carpet item group in favor of the rug item group, but didn't switch the references to the carpet group over.

Reply:
So I did. I actually didn't notice they were different.